unit_tests/test_redis_client_base.py

```diff
@@ -0,0 +1,179 @@
+import json
+import uuid
+import pytest
+from redis import StrictRedis
+import d3a_api_client
+from d3a_api_client.redis_client_base import RedisClientBase, RedisAPIException
+from unittest.mock import MagicMock
+
+area_id = str(uuid.uuid4())
+transactionID = str(uuid.uuid4())
+d3a_api_client.redis_client_base.StrictRedis = MagicMock(spec=StrictRedis)
+d3a_api_client.redis_client_base.StrictRedis.publish = MagicMock(spec=StrictRedis)
+d3a_api_client.redis_client_base.StrictRedis.pubsub.psubscribe = MagicMock()
+
+
+@pytest.fixture
+def redis_client_auto_register():
+    return RedisClientBase(area_id=area_id, autoregister=False)
+
+
+class TestRedisClientBase:
+
+    def test_function__subscribe_to_response_channels(self, redis_client_auto_register):
```

Comment:
```suggestion
    def test_subscribe_to_response_channels(self, redis_client_auto_register):
```

Reply:
The same comment (to remove `_function_`) applies to the entire PR (as I had written here: https://github.com/gridsingularity/d3a-api-client/pull/163#pullrequestreview-734068112)